sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -1025,7 +1025,7 @@ export class RuntimeExecutionTimes {
 }
 
 // @public
-export function setAuthorizationTokenHeaderUsingMasterKey(verb: HTTPMethod, resourceId: string, resourceType: ResourceType, headers: CosmosHeaders, masterKey: string): void;
+export function setAuthorizationTokenHeaderUsingMasterKey(verb: HTTPMethod, resourceId: string, resourceType: ResourceType, headers: CosmosHeaders, masterKey: string): Promise<void>;
```

Comment:
Updated to return a Promise since Browser's CryptoAPI is asynchronous. 
@southpolesteve, @bterlson - What do you think?

Reply:
I'll defer to @southpolesteve on this one :)

Reply:
Also, yay API Extractor for making this change super obvious!!

Reply:
This is fine. I believe we only exposed it for a portal use case where we do some weird stuff with auth headers